lib/site.js

```diff
@@ -239,6 +239,68 @@ Site.prototype.tag = function (slug) {
 };
 
 /**
+ * Get a list of shortcodes available on a site.
+ *
+ * Note: The current user must have publishing access.
+ *
+ * @api public
+ */
+
+Site.prototype.shortcodes = function (query, fn) {
```

Comment:
To be consistent we should rename this one to `shortcodesList` such as `postsList`, `usersList`, etc.